frontend/src/components/profile/ProfileHeader.tsx

```diff
@@ -10,7 +10,7 @@ type ProfileHeaderProps = {
 
 export default function ProfileHeader({ user, loading = false }: ProfileHeaderProps) {
   return (
-    <Stack direction="row" alignItems="center">
+    <Stack direction="row" alignItems={"center"}>
```

Comment:
i suggest keeping it as "center" (removing {}) to be consistent with others

Reply:
similar issues below too, please rectify accordingly

Reply:
Ok, done. Thanks